esphome/components/sensor/ppd42x.py

```diff
@@ -0,0 +1,53 @@
+import voluptuous as vol
+
+from esphome import pins
+from esphome.components import sensor
+import esphome.config_validation as cv
+from esphome.const import  CONF_ID, CONF_PM_10_0, CONF_PM_2_5, CONF_NAME, \
+    CONF_UPDATE_INTERVAL, CONF_TIMEOUT
+from esphome.cpp_generator import Pvariable, add
+from esphome.cpp_helpers import gpio_input_pin_expression, gpio_output_pin_expression, \
+    setup_component
+from esphome.cpp_types import App
+
+
+Ppd42xSensorComponent = sensor.sensor_ns.class_('Ppd42xSensorComponent',
+                                                sensor.PollingSensorComponent)
+
+
+PLATFORM_SCHEMA = cv.nameable(sensor.SENSOR_PLATFORM_SCHEMA.extend({
```

Comment:
```suggestion
PLATFORM_SCHEMA = sensor.PLATFORM_SCHEMA.extend({
```

Same as in -core, this class shouldn't publish both pm10 and pm2.5 in the same sensor.